corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -1079,16 +1092,30 @@ def initial_columns(self):
                 'sum': 'Sum',
                 'expand': 'Count per Choice'
             }
+            added_multiselect_columns = set()
             cols = []
             for c in self.existing_report.columns:
-                exists = self._column_exists(c['field'])
+                mselect_indicator_id = self._get_multiselect_indicator_id(
+                    c['field'], self.existing_report.config.configured_indicators
+                )
+                indicator_id = mselect_indicator_id or c['field']
+                display = c['display']
+                exists = self._column_exists(indicator_id)
+
+                if mselect_indicator_id:
+                    if mselect_indicator_id not in added_multiselect_columns:
+                        added_multiselect_columns.add(mselect_indicator_id)
+                        display = " - ".join(display.split(" - ")[:-1])
+                    else:
+                        continue
+
                 cols.append(
                     ColumnViewModel(
-                        display_text=c['display'],
+                        display_text=display,
                         exists_in_current_version=exists,
-                        property=self._get_column_option_by_indicator_id(c['field']).id if exists else None,
-                        data_source_field=c['field'] if not exists else None,
-                        calculation=reverse_agg_map.get(c.get('aggregation'), _('Count per Choice'))
+                        property=self._get_column_option_by_indicator_id(indicator_id).id if exists else None,
+                        data_source_field=indicator_id if not exists else None,
+                        calculation=reverse_agg_map.get(c.get('aggregation'), 'Count per Choice')
```

Comment:
Unfortunately the page doesn't use separate ids and display text for column format/aggregation type, so the value in this view model is both displayed to the user, and then potentially sent back to the server and used in [this dict](https://github.com/dimagi/commcare-hq/blob/a658108653116a0c7294ce15a4053f8ce906f410/corehq/apps/userreports/reports/builder/columns.py#L8), so things will actually break if the value is translated.

There are also likely other places where it is currently being translated but should not be. If, after going through the report builder prototyping process, we decide to keep this code, then we should switch to an id system.